src/pages/api/logs.ts

```diff
@@ -0,0 +1,24 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const referer = req.headers.referer;
+
+  const allowedReferers = ['https://www.moregoats.com/', 'https://dev.moregoats.com/'];
+
+  if (!referer || !allowedReferers.includes(referer)) {
+    res.status(403).json({ error: 'Unauthorized access' });
+    return;
+  }
```

Comment:
I believe that `referer` is not what we need here, i think that is something more like [Access-Control-Allow-Origin](https://developer.mozilla.org/es/docs/Web/HTTP/Headers/Access-Control-Allow-Origin)